src/classes/AFFL_AccRecordType_TDTM.cls

```diff
@@ -62,36 +71,47 @@ public class AFFL_AccRecordType_TDTM extends TDTM_Runnable {
          //    B. If that account did not have a record type:
          //         --> the key affl lookup field for that type in Contacts affiliated to it needs to be populated
     	 if(newlist != null && newlist.size() > 0 && oldlist != null && oldlist.size() > 0) {
-    	 	for (Integer i = 0; i < newlist.size(); i++) {
+    	 	
+             for (Integer i = 0; i < newlist.size(); i++) {
     	 	    SObject so = newlist[i];
     	 	    //AFTER UPDATE
     	 		if(triggerAction == TDTM_Runnable.Action.AfterUpdate) {
 	    	 		Account acc = (Account)so;
 			        Account oldAcc = (Account)oldList[i];
 
-			        if(acc.RecordTypeId != oldAcc.RecordTypeId) {
-			        	AFFL_MultiRecordTypeMapper afflMapper = new AFFL_MultiRecordTypeMapper();
+                    // If the record type isn't changing, move on
+                    if(acc.RecordTypeId == oldAcc.RecordTypeId) {
+                        continue;
+                    }
```

Comment:
Honestly I did it here because of our Apex PMD rules - the alert came up re: too many nested if statements. By short circuiting early, you can keep code cleaner because, in many cases, it results in less nesting :)

I'll think about the blog post idea...

Reply:
I just think it's something not everyone does. There's a stigma around things like continue and break but I think you're using them correctly here. Exactly what continue is for. :)